tests/acceptance/ContactManagementCest.php

```diff
@@ -0,0 +1,207 @@
+<?php
+
+class ContactManagementCest
+{
+    public function _before(AcceptanceTester $I)
+    {
+        $I->login('admin', 'Maut1cR0cks!');
+    }
+
+    public function createContactFromQuickAdd(AcceptanceTester $I)
+    {
+        // Navigate to the contacts page
+        $I->amOnPage('/s/contacts');
+
+        // Wait for and click on the "Quick Add" button
+        $I->waitForElementClickable('#toolbar .quickadd', 30);
+        $I->click('#toolbar .quickadd');
+
+        // Wait for the Quick Add modal to appear and verify its presence
+        $I->waitForElementVisible('#MauticSharedModal-label', 30);
+        $I->see('Quick Add', 'h4.modal-title');
+
+        // Wait for the form fields to be visible
+        $I->waitForElementVisible('#lead_firstname', 10);
+
+        // Fill out the quick add form
+        $I->fillField('#lead_firstname', 'QuickAddFirstName');
+        $I->fillField('#lead_lastname', 'QuickAddLastName');
+        $I->fillField('#lead_email', 'quickadd@example.com');
+        $I->fillField('#lead_tags_chosen input', 'TestTag');
+        $I->pressKey('#lead_tags_chosen input', Facebook\WebDriver\WebDriverKeys::ENTER);
```

Comment:
We should use `use Facebook\WebDriver\WebDriverKeys` at the top of the class and then simply use 
`WebDriverKeys::ENTER`